src/custom_commands_manager.ts

```diff
@@ -19,8 +20,10 @@ export class CustomCommandsManager implements Disposable, NeovimCommandProcessab
     public async handleVSCodeCommand(command: string, args: unknown[]): Promise<unknown> {
         const editor = window.activeTextEditor;
         if (!editor) return;
+        // Give some time to other possible events  to weak up
+        // 10ms basically has no side effects, even `cursorMove` which may be called frequently
+        await wait(10);
```

Comment:
This is fragile and not a good tradeoff. If the goal is to wait until input is "flushed" in the Nvim server, you can do that by calling a non-fast Nvim API function, e.g. `nvim_eval('')`.